install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -13,7 +13,11 @@ data:
   mesh: |-
     # Set the following variable to true to disable policy checks by the Mixer.
     # Note that metrics will still be reported to the Mixer.
+    {{- if .Values.mixer.policy.enabled }}
     disablePolicyChecks: {{ .Values.global.disablePolicyChecks }}
```

Comment:
nit.. shouldn't we simply set this to disablePolicyChecks: false and the other one to true?

Reply:
I think the idea is that you want to be able to turn on/off policy at runtime, without re-installing istio with/without policy.

Reply:
`mixer.policy.enabled` might be better thought of as `install.mixer.policy.config-and-deployments`.  as @kyessenov relates, this allows the installation of `istio-policy` while disabling its use by default.